### DISCUSSION TITLE
Add Chinese Simplified support

### DISCUSSION THREAD

**Discussion by Axia111** [Atlas-OS/Atlas#1318] [Suggestions]
Atlas is a very good system, many people in China are very willing to try this system, we very much hope to have official Chinese Simplified Chinese support, thank you very much!

**Comment by PostmanPat2011** [comment]
Run powershell as admin
Run
`Install-Language zh-Hans`
Wait for operation to complete
Run
`Set-SystemPreferredUILanguage zh-Hans`
Restart system. If this does not work try changing the language from the settings.
